metadata/README.md

```diff
@@ -0,0 +1,29 @@
+## Libphonenumber Metadata Tools
```

Comment:
We can wordsmith this later once things are being pushed auto-magically from the Google-side repository.